cdk/apps/rnasum/stacks/batch.py

```diff
@@ -0,0 +1,268 @@
+from aws_cdk import (
```

Comment:
Would it be possible to encapsulate `batch.py` from umccrise app on a python object/class and reuse it here instead of replicating most of it? Same goes for `cicd`... it should be possible to generalise and reuse for both apps, umccrise and rnasum.

Reply:
Agreed, but happy to get it deployed and working first.

Reply:
Yap; agree. Could do TF `module` alike re-usable common code here. Need to refactor a bit on app code structure itself, so that can utilise Python package and modules for inheritance and generalisation. Could do improvement in next iteration.